content/news/release-20180307.md

```diff
@@ -0,0 +1,76 @@
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+We are today pushing a new bugfix release that addresses a collection of issues that were missed during the playtesting phase of release-20180218.
+
+The changes in release-20180307 are:
+
+* Fixed an issue that could cause excessive debug log file sizes.
+* Fixed a compatibility issue with mono 5.8 and newer.
+* Increased the minimum required mono version to 4.2.
```

Comment:
It would be good to emphasise that this is to fix the "Failed to query server list" issue.

Reply:
Done locally, will push it with the .rb update later.